playground/nextjs-app-router/components/AppProvider.tsx

```diff
@@ -164,7 +188,15 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
         setDefaultMaxSlippage,
       }}
     >
-      {children}
+      <OnchainKitProvider
+        apiKey={ENVIRONMENT_VARIABLES[ENVIRONMENT.API_KEY]}
+        chain={base}
+        config={{ theme: componentTheme }}
+        projectId={ENVIRONMENT_VARIABLES[ENVIRONMENT.PROJECT_ID]}
+        schemaId="0xf8b05c79f090979bf4a80270aba232dff11a10d9ca55c4f88de95317970f0de9"
+      >
+        {children}
+      </OnchainKitProvider>
```

Comment:
I know this is strange. It was the only way I was able to get the componentSwitcher to work in playground. 

Updating the OnchainKitProvider directly caused strange React rendering errors. 

I've implemented this solution to unblock QA and testing in general